frontend/src/components/Profile/ProfileCard.vue

```diff
@@ -2,28 +2,45 @@
   <v-card :color="color" class="ma-y8" elevation="0">
     <v-list-item>
       <v-list-item-avatar>
-        <v-img :src="profile.picture" :alt="profile.pseudo" />
+        <v-img :src="picture" alt="profile picture" />
       </v-list-item-avatar>
       <v-list-item-content>
-        {{ profile.pseudo }}
+        {{ username() }}
       </v-list-item-content>
     </v-list-item>
   </v-card>
 </template>
 
 <script>
+import { mapGetters } from "vuex";
+
 export default {
+  data: function () {
+    return {
+      ...mapGetters(["username", "avatar", "id"]),
+    };
+  },
+  computed: {
+    picture() {
+      if (
+        this.$store.state.avatar === undefined ||
+        this.$store.state.avatar === null
+      ) {
+        return require("@/assets/images/king-pong.png");
+      }
+      return this.$store.state.avatar;
```

Comment:
Would it be possible to update the `getter` for `avatar` or `getAvatar` to return `king-pong.png` if no avatar is found? That would mean that we wouldn't have to check everywhere that we use a user's avatar. (This would probably depend on my comment about refactoring getAvatar)

Reply:
I agree, but let us do it in a separate issue. See #116